presto-docs/src/main/sphinx/security/cli.rst

```diff
@@ -0,0 +1,118 @@
+===========================
+CLI Kerberos Authentication
+===========================
+
+The Presto :doc:`/installation/cli` can connect to a :doc:`Presto coordinator
+</security/server>` that has Kerberos authentication enabled.
+
+Environment Configuration
+-------------------------
+
+.. |subject_node| replace:: client
+.. include:: kerberos-services.fragment
+.. include:: kerberos-configuration.fragment
+
+Kerberos Principals and Keytab Files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Each user who connects to the Presto coordinator needs a Kerberos principal.
+You will need to create these users in Kerberos using `kadmin
+<http://web.mit.edu/kerberos/krb5-latest/doc/admin/admin_commands/kadmin_local.html>`_.
+
+Additionally, each user needs a `keytab file
+<http://web.mit.edu/kerberos/krb5-devel/doc/basic/keytab_def.html>`_. The
+keytab file can be created using :command:`kadmin` after you create the
+principal.
+
+.. code-block:: none
+
+  kadmin
+  > addprinc -randkey someuser@EXAMPLE.COM
+  > ktadd -k /home/someuser/someuser.keytab someuser@EXAMPLE.COM
+
+.. include:: ktadd-note.fragment
+
+.. include:: jce-policy.fragment
+
+Java Keystore File for TLS
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Access to the Presto coordinator should be through https when using Kerberos
+authentication. The Presto coordinator uses a :ref:`Java Keystore
+<coordinator_java_keystore>` file for its TLS configuration. This file can be
+copied to the client machine and used for its configuration.
+
+Presto CLI execution
+--------------------
+
+Invoking the CLI with Kerberos support enabled requires a number of command
```

Comment:
Suggest:
In addition to the options that are required when connecting to a Presto coordinator that does not require Kerberos authentication, invoking the CLI with Kerberos support enabled requires a number of command
 line options.
 Presto coordinator that does not require Kerberos authentication.